packages/backend/test/peripherals/database/KeyValueStore.test.ts

```diff
@@ -16,21 +16,22 @@ describe(KeyValueStore.name, () => {
     kvStore.set('key', 'value')
     const actual = await kvStore.get('key')
     expect(actual).toEqual('value')
+    await kvStore.delete('key')
   })
 
   it('reads and removes all values', async () => {
-    await Promise.all([
```

Comment:
I don't understand. As long as you await knex's promise in the repository it should be fine. Why would Promise.all not work?